Assets/Scripts/CountdownTimer.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+using System;
+//not finished, need a visualization for the timer (ex. text)
+// countdown timer for the stage
+// once the timer reaches 0.0 sec activate fail state
+
+public class CountdownTimer : MonoBehaviour
+{
+    [SerializeField] private float GameTime;
+    [SerializeField] public Text num ;
```

Comment:
public fields are serialized by default, so it is not necessary to have this [SerializeField] attribute